enderio-base/src/main/java/com/enderio/base/common/enchantments/AutoSmeltEnchantment.java

```diff
@@ -0,0 +1,39 @@
+package com.enderio.base.common.enchantments;
+
+import javax.annotation.Nonnull;
+
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.item.enchantment.Enchantment;
+import net.minecraft.world.item.enchantment.EnchantmentCategory;
+import net.minecraft.world.item.enchantment.Enchantments;
+
+public class AutoSmeltEnchantment extends EIOBaseEnchantment{
+
+	//config rarity
```

Comment:
Could you prefix all of these `config` markers with TODO so they show up in IDEs?